README.md

```diff
@@ -25,14 +25,14 @@ You must have [Docker Engine](https://docs.docker.com/install/) installed to
 build the Splunk Operator.
 
 This project uses [Go modules](https://blog.golang.org/using-go-modules),
-and requires [golang](https://golang.org/doc/install) 1.17.3 or later.
+and requires [golang](https://golang.org/doc/install) 1.13 or later.
```

Comment:
This is undoing https://github.com/splunk/splunk-operator/pull/581, please update back to 1.17.3
cc: @pdhanoyasplunk

Reply:
Fixed